modules/consensus.py

```diff
@@ -230,13 +229,19 @@ def polish_sequences(centers, args):
             print("running medaka on spoa reference {0} using {1} reads for polishing.".format(c_id, nr_reads_used))
             # for (nr_reads_in_cluster, c_id, spoa_center_file, all_reads_file) in to_polishing:
             polishing_outfolder = os.path.join(args.outfolder, "medaka_cl_id_{0}".format(c_id))
+            outfiles = [  # consider all output formats for compatibility with all Medaka versions
+                os.path.join(polishing_outfolder, "consensus.fasta"),
+                os.path.join(polishing_outfolder, "consensus.fastq")
+            ]
             help_functions.mkdir_p(polishing_outfolder)
-            run_medaka(all_reads_file, spoa_center_file, polishing_outfolder, "1", args.medaka_model)
-            print("Saving medaka reference to file:", os.path.join(args.outfolder, "medaka_cl_id_{0}/consensus.fasta".format(c_id)))   
-            with open(os.path.join(polishing_outfolder, "consensus.fasta"), 'r') as cf:
-                l = cf.readlines()
-            center_polished = l[1].strip()
-            centers[i][2] = center_polished
+            run_medaka(all_reads_file, spoa_center_file, polishing_outfolder, "1", args.medaka_model, outfastq=args.medaka_fastq)
+            print("Saving medaka reference to file:", os.path.join(polishing_outfolder, "consensus.fasta/q"))
+            for f in outfiles:
```

Comment:
This part is testing if a .fasta or .fastq file has been generated by Medaka, even if the -q was used. The idea is to be compatible with all versions of Medaka. Indeed since the introduction of the -q feature, Medaka had a minor bug (https://github.com/nanoporetech/medaka/issues/464) and was still producing a file suffixed as .fasta.